platform/linux-generic/include/odp/api/plat/event_inlines.h

```diff
@@ -54,6 +56,11 @@ _ODP_INLINE int odp_event_type_multi(const odp_event_t event[], int num,
 	return i;
 }
 
+_ODP_INLINE uint32_t odp_event_flow_id(odp_event_t event)
+{
+	return _odp_event_hdr_field(event, int8_t, flow_id);
```

Comment:
`flow_id` seems to be of type `uint8_t`.

Reply:
Fixed in V2.